app/models/katello/content_view_version.rb

```diff
@@ -294,11 +294,11 @@ def docker_manifest_count
     end
 
     def docker_tag_count
-      ::Katello::DockerMetaTag.where(:repository_id => repositories.archived.docker_type).count
+      ::Katello::DockerMetaTag.where(:id => RepositoryDockerMetaTag.where(:repository_id => repositories.archived.docker_type).select(:docker_meta_tag_id)).count
```

Comment:
could you call docker_tags.count ?

Reply:
I'll give it a try!

Reply:
ok, I've changed it.  Works at first testing but we'll see what the Jenkins run says